packages/server/lib/errors.coffee

```diff
@@ -842,6 +842,18 @@ getMsgByType = (type, arg1 = {}, arg2) ->
 
       Please do not modify CYPRESS_ENV value.
       """
+    when "CDP_COULD_NOT_CONNECT"
+      """
+      Cypress could not make a connection to the Chrome DevTools Protocol after 5 seconds.
+
+      Without this, your tests can not run. Please check your Chrome installation and try again.
```

Comment:
Cypress failed to make a connection to the Chrome DevTools Protocol after retrying for 5 seconds.

This usually indicates there was a problem opening the Chrome browser.